_posts/2015-04-20-feedhenry-push-templates.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
@lfryc here is a PR for a newe date:
https://github.com/edewit/aerogear.org/pull/4